### DISCUSSION TITLE
42회차 반성문

### DISCUSSION THREAD

**Discussion by amaran-th** [BDD-CLUB/challenge-every-week-posting#418] [6. 반성문]
안녕하세요...반성문보다는 근황토크가 됐는데
사실 이번에 번아웃이 세게 와서 개발에 대한 의욕을 완전히 잃었습니다
그래서 요즘 글을 쓸 주제도 없네요...당분간은 회복이 될 때까지 휴식을 취해야 할 것 같습니다.
챌린지 운영자로서 역할을 하지 못하고 있는 것 같아 마음이 좋지 않네요🥲
tmi지만 새로운 걸 배워보면 나아질까 싶어서 UI/UX 디자인을 배우고 있습니다. 생각보다 굉장히 재밌더라구요??
조만간 진로가 바뀔지도 모르겠습니다(ㅋㅋㅋ)

**Comment by stopmin** [comment]
파이팅입니닷 아마란스!! 항상 응원합니다💪
